src/app/core/models/content-pagelet-entry-point/content-pagelet-entry-point.mapper.ts

```diff
@@ -43,6 +44,7 @@ export class ContentPageletEntryPointMapper {
       resourceSetId: data.resourceSetId,
       pageletIDs,
       configurationParameters,
+      seoAttributes: SeoAttributesMapper.fromCMSData(pagelets.length && pagelets[0]),
```

Comment:
It would be good to add the seoAttributes only for content pages and not for content includes.

Suggestion: Add content entrypoint `page` flag if fetched via `/page` REST resource and use this to distinguish if to apply the `SeoAttributesMapper.fromCMSData` or not.

Reply:
Hmm, then I'd rather do it outside altogether. I also didn't really like this approach, as it mixes concerns. 😉